### ISSUE TITLE
 'dashboardHeaderPlus' not exported by 'namespace:shinydashboardPlus'

### ISSUE THREAD

**Issue by paladinic** [Tychobra/polishedpayments#9]
I am seeing the error below preventing me from installing.
I believe it might be related to[ this issue](https://github.com/RinteRface/shinydashboardPlus/issues).

```
Error: object 'dashboardHeaderPlus' is not exported by 'namespace:shinydashboardPlus'
Execution halted
ERROR: lazy loading failed for package 'polishedpayments'
```

**Comment by nathhenry3** [comment]
It looks like the shinydashboardPlus package has been updated to v2 recently, and they've removed the 'dashboardHeaderPlus' object. 

I talked to one of the package maintainers, they say they're going to start development again in a few weeks as they're busy with other projects.

**Comment by merlinoa** [comment]
Sorry for the delay here.  This should be fixed with ac51d6e07a9d14788bd3353869849dc151c58fa0 .  Please reopen if you run into any issues.
